packages/orm/src/EntityManager.ts

```diff
@@ -541,6 +543,16 @@ export class EntityManager<C = {}> {
     return entities as T[];
   }
 
+  /**
+   * Loads entities found, when starting at `entities`, via the "path" given by the `fn` lens function.
+   *
+   * Results are unique, i.e. if doing `em.loadLens([b1, b2], b => b.author.publisher)` point to the
+   * same `Publisher`, it will only be returned as a single value.
+   */
+  public async loadLens<T extends Entity, U, V>(entities: T[], fn: (lens: Lens<T>) => Lens<U, V>): Promise<V> {
```

Comment:
I don't love the name `loadLens`; I'd originally thought about overloading `loadAll` like:

```
const pubs = await em.loadAll(books, b => b.author.publisher);
```

But we've already got a `loadAll` that is `loadAll(Book, ids)` and it seemed like this was too different to be an overload.